faker/providers/address/fr_FR/__init__.py

```diff
@@ -467,3 +466,12 @@ def department_number(self) -> str:
         :example: '59'
         """
         return self.department()[0]
+
+    def postcode(self) -> str:
+        """
+        Randomly returns a postcode generated from existing french depertment number.
+
+        exemple: '33260'
+        """
+        department = self.department()
```

Comment:
did you mean to call `self.department_number()`?

Reply:
:facepalm: